locations/Bahraich/inputs/simulation/clinic.yaml

```diff
@@ -0,0 +1,59 @@
+---
+################################################################################
+# clinic.yaml - Parameters for specifying a health centre clinic within CLOVER #
+#                                                                              #
+# Author: Ilaria Del Frate                                                     #
+# Copyright: laria Del Frate, 2022                                             #
+# Date created: 27/07/2022                                                     #
+# License: Open source                                                         #
+################################################################################
+
+clinics:
+  # A dash here means one entry, i.e., clinic.
+  - name: kabakedi
+    # Transmission Load
+    floor_area: 56.49 # [m^2]
+    roof_area: 56.49 # [m^2]
+    surface_area_walls: 86.8 # [m^2]
+
+    # Assumption of 2 the open windows covered with glass
+    surface_area_doors_windows: 15.13 # [m^2]
+    u_value_walls: 2.5 # [W/m^2*K] Thermal transmittance of wall
+    u_value_windows_doors: 5 # [W/m^2*K]
+    u_value_floor: 1 # [W/m^2*K]
+    u_value_roof: 0.7 # [W/m^2*K]
+    inside_ideal_temperature: 23 # [ °C]
+    # outside_temperature: 28 #[ °C]
+
+    # Internal heat Load - People
+    people_1: 1.5 #number of nurses present per day
+    people_2: 2.5 #average number of patients and accompaniers
+    heat_loss_people_1: 82 # [W] From Drug Store ASHRAE value
+    heat_loss_people_2: 67.4 # [W] From Apartments ASHRAE value
+    start_time_people_1: [11, 11, 11, 11, 11, 11, 0]
+    end_time_people_1: [18, 18, 18, 18, 18, 18, 0]
+    start_time_people_2: [12, 12, 12, 12, 12, 12, 0]
+    end_time_people_2: [17, 17, 17, 17, 17, 17, 0]
+
+    # Internal heat load - Lighting
+    lamps_internal: 9 # number of lamps within the clinic
+    lamps_external: 4 # number of lamps outside the clinic
+    start_time_lamps: [15, 15, 15, 15, 15, 15, 0] # At 15:00 lights are turned on
+    end_time_lamps: [18, 18, 18, 18, 18, 18, 0]
+    time_lamps_external: 12 # hours per day (night)
+    wattage_lamps: 7 # [W] LED Bulb light A-8901
+
+    # Equipment heat load
+    fridge_wattage: 120 # [W]
+    time_fridge: 24 # [h/day]
```

Comment:
I left the fridge in the assumption form because I knew that there are already fridge calculations on CLOVER that can be  useful. The fridge is opened and closed during the working hours, and mostly between 12 to 17, while during the night and morning is closed all the time. I have the fridge model and specifications and I am not really sure on the value I should use for the heat dissipation, but I can show it to you tomorrow maybe.